Formula/circleci.rb

```diff
@@ -0,0 +1,124 @@
+class Circleci < Formula
+  desc "CircleCI Local CLI"
+  homepage "https://circleci.com/docs/2.0/local-cli/"
+  url "https://github.com/circleci/local-cli/releases/download/v0.0.4705-deba4df/circleci_v0.0.4705-deba4df.tar.gz"
```

Comment:
> The URL https://github.com/circleci/local-cli/releases/download/v0.0.4705-deba4df/circleci_v0.0.4705-deba4df.tar.gz is not reachable (HTTP status code 404)

Reply:
The PR on the other end wasn't merged yet. It is not, the GitHub Release made, and this URL now works.